tools/Customization/DevHome.Customization/Views/OptimizeDevDriveDialog.xaml

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright (c) Microsoft Corporation.. -->
+<!-- Licensed under the MIT License. -->
+
+<ContentDialog
+    x:Class="DevHome.Customization.Views.OptimizeDevDriveDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:environments="using:DevHome.Customization.ViewModels.Environments" d:DataContext="{d:DesignInstance Type=environments:DevDriveOptimizerCardViewModel}"
+    x:Uid="ms-resource:///DevHome.Customization/Views/OptimizeDevDriveDialog"
+    mc:Ignorable="d"
+    CornerRadius="8"
+    Title="Choose directory on dev drive..."
+    PrimaryButtonText="Make changes"
+    SecondaryButtonText="Cancel"
+    PrimaryButtonClick="OnDirectoryInputConfirmed"
+    SecondaryButtonClick="OnDirectoryInputCancelled">
+    <StackPanel>
+        <TextBlock TextWrapping="Wrap">
+            <Run Text="{x:Bind OptimizeDevDriveDialogDescription, Mode=OneWay}"/>
+        </TextBlock>

```

Comment:
Better to just bind the text of the TextBlock
```suggestion
        <TextBlock Text="{x:Bind OptimizeDevDriveDialogDescription, Mode=OneWay}" TextWrapping="Wrap" />
```
BUT this actually needs to be localized, so you need to add the description you want to a resw file with a Name like `OptimizeDevDriveDialogDescription.Text` and refer to it here like this:
```suggestion
        <TextBlock x:Uid="{x:Bind OptimizeDevDriveDialogDescription, Mode=OneWay}" TextWrapping="Wrap" />
```